topics/epigenetics/tutorials/ewas-suite/tutorial.md

```diff
@@ -1,38 +1,50 @@
 ---
 layout: tutorial_hands_on
 
-title: "EWAS data analysis of 450k data"
+title: "Infinium Human Methylation BeadChip"
 zenodo_link: "https://zenodo.org/record/1251211#.WwREQ1Mvz-Y"
 questions:
-  - "What is a epigenome-wide association studies?"
-  - "Why is a EWAS analysis useful?"
-  - "Why 450k data?"
+  - "Which DNA regions and positions are diffrentialy methylated in pre MAPKi treatment and post MAPKi resistance Melanomas GSE65183?"
+  - "How to analyse and visualise Infinium Human Methylation BeadChip's?"
```

Comment:
is it really just one hour? :)

Reply:
yes, as it is a small subset of the original data